packages/framework/esm-styleguide/src/workspaces/workspaces.ts

```diff
@@ -95,6 +96,17 @@ export interface WorkspaceStoreState {
 
 export interface OpenWorkspace extends WorkspaceRegistration, DefaultWorkspaceProps {
   additionalProps: object;
+  /**
+   * The workspace family store is a store that is specific to the workspace sidebar family.
+   * If the workspace has its own sidebar, the store will be created and passed to the workspace.
+   * This store can be used to store data that is specific to the workspace sidebar family.
+   * The store will be same for all the workspaces with same sidebar family name.
+   *
+   * For workspaces with no sidebarFamilyName or sidebarFamilyName as 'default', the store will be undefined.
+   *
+   * The store will be cleared when all the workspaces with the store's sidebarFamilyName are closed.
+   */
+  workspaceFamilyStore?: StoreApi<object> | undefined;
```

Comment:
```suggestion
  workspaceFamilyStore?: StoreApi<object>;
```

Seems redundant with the `?`, right?